code/commands/view-commands.cpp

```diff
@@ -504,7 +504,11 @@ void Gobby::ViewCommands::on_menu_language_changed()
 
 		// The language should exist by construction, if the languages
 		// available in the language manager don't change at runtime
-		g_assert(language != NULL);
+		if(language != NULL) {
+			g_warning("Language \"%s\" is not known to GtkSourceView",
+				language_id.c_str());
+			return;
```

Comment:
Are you sure you want to return from the function here? `NULL` is a perfectly legal value for `language` in this case, as can be seen by the initial value of `language` which will be used as-is when the `language_id.empty()` condition above is false.

Perhaps simply taking out `g_assert(language != NULL);` would suffice, and the user is not going to look at the GLib warning logs anyway.